examples/display_lvgl_demos/README.md

```diff
@@ -2,32 +2,64 @@
 
 This example shows LVGL internal demos with RGB LCD.
 
-### Configurations
-
-To improve display performance (FPS), please set the following configurations:
-
-* System:
-    * `ESP_DEFAULT_CPU_FREQ_MHZ_240`
-    * `FREERTOS_HZ` = 1000
-    * `COMPILER_OPTIMIZATION_PERF`
-* Flash:
-    * `ESPTOOLPY_FLASHMODE_QIO`
-    * `ESPTOOLPY_FLASHFREQ_120M`
-* PSRAM:
-    * `SPIRAM_MODE_OCT`
-    * `SPIRAM_SPEED_120M` (See [here](https://github.com/espressif/esp-dev-kits/tree/master/esp32-s3-lcd-ev-board#psram-120m-ddr) to enbale this feature of ESP-IDF)
-    * `SPIRAM_FETCH_INSTRUCTIONS`
-    * `SPIRAM_RODATA`
-* Cache:
-    * `ESP32S3_DATA_CACHE_LINE_64B` (It can be enabled only when using bounce buffer or PSRAM with Octal 120M. Otherwise it will cause screen drift.)
```

Comment:
Why is this note deleted?

Reply:
The info here is redundant and already present in the existing [docs](https://docs.espressif.com/projects/esp-faq/en/latest/software-framework/peripherals/lcd.html#how-can-i-improve-the-display-frame-rate-of-lcd-screens). I think we shouldn't maintain two of them. 

But it's better to add info that guides users to find it, so I added a link in README. What do you think?